src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceAppDomainTests.cs

```diff
@@ -37,6 +37,8 @@ public Exception LoadAnalyzer(string analyzerPath)
 
     public class AnalyzerFileReferenceAppDomainTests : TestBase
     {
+        private const int MaxWarningLevel = 9999;
```

Comment:
Because this project doesn't have a reference to `TestOptions`, I created this const. But I don't feel this approach is good. This const is defined in a few other tests in this project.

Reply:
Perhaps it would be reasonable to define it next to

https://github.com/dotnet/roslyn/blob/897d928ff76cc92ef6aa5255f0a78da32464fbc0/src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs#L29

Reply:
Why isn't max warning level just `int.MaxValue` ?

Reply:
It should work, but it was documented somewhere as 9999 so I matched that.